Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -791,6 +822,17 @@
 		                fileName = [x.S.output_ID(1:end-16) '_ROI' xASL_num2str(iROI) '-' namesROIuse{iROI} '_' x.S.SubjectSessionID{SubjSess,1} '_CBF'];
                         [pathOutput_CBF] = xASL_stat_VisualizeSubjectWiseROI(x, xASL_im_Column2IM(CurrentMaskVascular, x.S.masks.WBmask), xASL_im_Column2IM(DataIm, x.S.masks.WBmask), fileName, pathOutput_CBF);
 
+                        % Check if the ROI size is large enough
+                        imMask = CurrentMaskVascular;
+                        imMask = (imMask>0) & isfinite(DataIm);
+                        imMask = imMask & (DataIm~=0); % Exclude zero values as well
+    
+                        if sum(imMask)<MinVoxels
+                            fprintf('\n');
+                            warning('%s\n', [x.S.TissueMaskingLocal ' ' namesROIuse{iROI} ' only contains ' xASL_num2str(sum(imMask)) ' voxels, so this ROI will be skipped in CBF calculations']);

```

Comment:
same as above

Reply:
Done in 46c421bcd1aed8156cbe146a8cf708930de3d337